<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SCTaskWorkModeTaskCodeSelectWidget</class>
 <widget class="QWidget" name="SCTaskWorkModeTaskCodeSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>任务代号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>点频</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="scTargetRadioBtn1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="scTaskCodeCombo1"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="scPointFreqCombo1"/>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="scTargetRadioBtn2">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="scTaskCodeCombo2"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="scPointFreqCombo2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
